src/commands/init.ts

```diff
@@ -0,0 +1,65 @@
+import { spawn } from 'child_process';
+import yargs from 'yargs';
+import which from 'which';
+import { log } from '../log.js';
+import { CommonProps } from '../types.js';
+import { sendError } from '../analytics.js';
+
+export const command = 'init';
+export const describe =
+  'Initialize a new Neon project using your AI coding assistant';
+export const builder = (yargs: yargs.Argv) =>
+  yargs
+    .option('context-file', {
+      hidden: true,
+    })
+    .strict(false);
+
+export const handler = async (
+  args: CommonProps & {
+    '--'?: string[];
+  },
+) => {
+  const passThruArgs = args['--'] || [];
+  await runNeonInit(passThruArgs);
+};
+
+const runNeonInit = async (args: string[] = []) => {
+  const npxPathOrNull = await which('npx', { nothrow: true });
+
+  if (npxPathOrNull === null) {
+    const error = new Error('npx is not available in the PATH');
+    log.error(error.message);
+    sendError(error, 'NPX_NOT_FOUND');
+    process.exit(1);
+  }
+
+  const isWindows = process.platform === 'win32';
+
+  const child = isWindows
+    ? spawn('cmd', ['/c', 'npx', 'neon-init', ...args], {
+        stdio: 'inherit',
+      })
+    : spawn(npxPathOrNull, ['neon-init', ...args], {
+        stdio: 'inherit',
+      });
+
+  for (const signame of ['SIGINT', 'SIGTERM']) {
+    process.on(signame, (code) => {
```

Comment:
the argument name `code` is misleading, it's actually the same signal name, so it's not even necessary